### PR TITLE
Add pointers back to Data Integrity Privacy and Security Considerations sections

### PR DESCRIPTION
**Pull request by msporny**
This PR attempts to address issue #29, raised by the PING and security review, by pointing back to the Data Integrity Security and Privacy Considerations section.

/cc @kdenhartog

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-di-ecdsa/pull/33.html" title="Last updated on Aug 26, 2023, 6:44 PM UTC (ab64324)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-di-ecdsa/33/a9f2510...ab64324.html" title="Last updated on Aug 26, 2023, 6:44 PM UTC (ab64324)">Diff</a>

### CODE REVIEW

**Review by iherman — APPROVED** [review]

**Review by dlongley — APPROVED** [review]

**Review by Wind4Greg — APPROVED** [review]

**Review by kdenhartog — COMMENTED** [review]
Can we reference the vc-data-model spec as well? There's additional points in there that are worth considering in here and it may (if the WG is fine with listing once in data model spec) reduce the number of considerations that need to be redundantly placed across all specs related to the same problems.

File: index.html [hunk 1]
```diff
@@ -2527,32 +2527,41 @@ <h4>Verify Derived Proof (ecdsa-sd-2023)</h4>
...
+<a href="https://www.w3.org/TR/vc-data-integrity/#security-considerations">
+Security Considerations section of the Data Integrity specification</a>.
+      </p>
+
       <p>
-The security (integrity/authenticity) of a verifiable credential
-signed by a digital signature algorithm is dependent on a number of factors
-including:
+The integrity and authenticity of a secured document that is protected by
+this cryptographic suite is dependent on a number of factors including:
```
Comment (TallTed) [inline index.html:2540]:
```suggestion
this cryptographic suite is dependent on a number of factors including
the following:
```

File: index.html [hunk 2]
```diff
@@ -2527,32 +2527,41 @@ <h4>Verify Derived Proof (ecdsa-sd-2023)</h4>
...
       </p>
+
       <ul>
         <li>
-the correct application of the signature algorithm to a verifiable
-credential (this specification)
+the correct application of a digital signature to a document,
```
Comment (TallTed) [inline index.html:2545]:
```suggestion
correct application of a digital signature to the document
```

File: index.html [hunk 3]
```diff
@@ -2527,32 +2527,41 @@ <h4>Verify Derived Proof (ecdsa-sd-2023)</h4>
...
         </li>
         <li>
-the choice of signature algorithm (ECDSA) and its parameters (P-256, P-384)
+the choice of signature algorithm (ECDSA) and its parameters (P-256, P-384),
```
Comment (TallTed) [inline index.html:2548]:
```suggestion
choice of an appropriate signature algorithm (ECDSA) and its parameters
(P-256, P-384)
```

File: index.html [hunk 4]
```diff
@@ -2527,32 +2527,41 @@ <h4>Verify Derived Proof (ecdsa-sd-2023)</h4>
...
         </li>
         <li>
-the correct implementation and usage of the signature algorithm,
-particularly with respect to well-known problem areas
+the correct implementation and usage of the digital signature algorithm,
+particularly with respect to well-known problem areas, and
```
Comment (TallTed) [inline index.html:2552]:
```suggestion
correct implementation and usage of the digital signature algorithm,
particularly with respect to well-known problem areas
```

File: index.html [hunk 5]
```diff
@@ -2527,32 +2527,41 @@ <h4>Verify Derived Proof (ecdsa-sd-2023)</h4>
...
         </li>
         <li>
 the proper management of the private and public keys used for
```
Comment (TallTed) [inline index.html:2555]:
```suggestion
proper management of the private and public keys used for
```

File: index.html [hunk 6]
```diff
@@ -2527,32 +2527,41 @@ <h4>Verify Derived Proof (ecdsa-sd-2023)</h4>
...
-signing and verification
+signing and verification.
```
Comment (TallTed) [inline index.html:2556]:
```suggestion
signing and verification
```

**Review by TallTed — COMMENTED** [review]
Grammar and punctuation fixes

**Comment by msporny** [comment]
> Can we reference the vc-data-model spec as well? There's additional points in there that are worth considering in here and it may (if the WG is fine with listing once in data model spec) reduce the number of considerations that need to be redundantly placed across all specs related to the same problems.

This specification (and soon, all of the cryptosuite specifications), ask the reader to consider the Security and Privacy consideration sections in the Data Integrity specification, which then asks the reader to consider the Security and Privacy consideration sections in the Verifiable Credentials specification. While we /could/ repeat the same language that's in the Data Integrity specification (that tells the reader to consider the Security and Privacy Considerations for VCs), we'd be duplicating guidance (with, arguably, not much more of an effect).

So, in the name of reducing the duplication of guidance, I'm suggesting that just pointing back to DI spec, which points back to the VC spec, is enough.

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, justification provided for not making one change, no objections, merging.

**Pull request merged**